# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/psmisc/use-either-ncurses-or-termcap.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/pstree.c.orig	2004-08-29 17:20:07.000000000 +0200
+++ ./src/pstree.c	2004-08-29 17:11:26.000000000 +0200
@@ -16,9 +16,12 @@
 #include <pwd.h>
 #include <dirent.h>
 #include <curses.h>
-#include <term.h>
 #include <termios.h>
+#ifdef HAVE_TERMCAP_H
 #include <termcap.h>
+#else
+#include <term.h>
+#endif
 #include <langinfo.h>
 #include <sys/types.h>
 #include <sys/stat.h>
